app/views/partners/profiles/edit.html.erb

```diff
@@ -1,3 +1,10 @@
+<script type="text/javascript">
+    document.onreadystatechange = function(){//window.addEventListener('readystatechange',function(){...}); (for Netscape) and window.attachEvent('onreadystatechange',function(){...}); (for IE and Opera) also work
+        if(document.readyState=='loaded' || document.readyState=='complete')
+            calculate_client_share_total();
+    }
```

Comment:
See above - we can attach a Stimulus controller and have the calculation happen on `connect()`.